src/test/kotlin/org/jetbrains/research/ml/coding/assistant/system/finder/VertexFinderTest.kt

```diff
@@ -0,0 +1,69 @@
+package org.jetbrains.research.ml.coding.assistant.system.finder
+
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.components.service
+import com.intellij.psi.PsiFile
+import org.jetbrains.research.ml.coding.assistant.dataset.TaskTrackerDatasetFetcher
+import org.jetbrains.research.ml.coding.assistant.dataset.model.DatasetTask
+import org.jetbrains.research.ml.coding.assistant.dataset.model.MetaInfo
+import org.jetbrains.research.ml.coding.assistant.solutionSpace.Util
+import org.jetbrains.research.ml.coding.assistant.solutionSpace.builder.SolutionSpaceGraphBuilder
+import org.jetbrains.research.ml.coding.assistant.solutionSpace.utils.psiCreator.PsiCreator
+import org.jetbrains.research.ml.coding.assistant.solutionSpace.weightCalculator.CustomEdgeWeightCalculator
+import org.jetbrains.research.ml.coding.assistant.system.PartialSolution
+import org.jetbrains.research.ml.coding.assistant.system.matcher.EditPartialSolutionMatcher
+import org.jetbrains.research.ml.coding.assistant.unification.CompositeTransformation
+import org.jetbrains.research.ml.coding.assistant.unification.DatasetUnification
+import org.jetbrains.research.ml.coding.assistant.util.ParametrizedBaseWithSdkTest
+import org.junit.Ignore
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+import java.io.File
+
+@Ignore
```

Comment:
Why do you ignore this test?

Reply:
a little long to compute

Reply:
Maybe it's better to create different tasks to run some tests locally so you can exclude some tests like this
```
test {
    exclude 'org/boo/**'
    exclude '**/Bar.class'
}
```
and on CI (so it doesn't matter how much time it takes to compute)